be/src/types/hll.cpp

```diff
@@ -445,7 +513,7 @@ int64_t HyperLogLog::estimate_cardinality() const {
     int num_zero_registers = 0;
 
     for (int i = 0; i < HLL_REGISTERS_COUNT; ++i) {
-        harmonic_mean += powf(2.0f, -_registers.data[i]);
+        harmonic_mean += harmomic_tables[_registers.data[i]];
```

Comment:
Is it possible that `_registers.data[i]` > 64?

Reply:
_registers.data[i]  <= 40 always holds